docs/resources/mws_workspaces.md

```diff
@@ -339,12 +340,14 @@ You can specify a `token` block in the body of the workspace resource, so that T
 * `comment` - (Optional) Comment, that will appear in "User Settings / Access Tokens" page on Workspace UI. By default it's "Terraform PAT".
 * `lifetime_seconds` - (Optional) Token expiry lifetime. By default its 2592000 (30 days).
 
+### Updating workspaces
+
 On AWS, the following arguments could be modified after the workspace is running:
 
-* `network_id` - (Optional, AWS only) `network_id` from [networks](mws_networks.md). Modifying [networks on running workspaces](mws_networks.md#modifying-networks-on-running-workspaces) would require three separate `terraform apply` steps.
-* `credentials_id` - (AWS only) `credentials_id` from [credentials](mws_credentials.md)
-* `storage_customer_managed_key_id` - (Optional, AWS only) `customer_managed_key_id` from [customer managed keys](mws_customer_managed_keys.md) with `use_cases` set to `STORAGE`. This is used to encrypt the DBFS Storage & Cluster EBS Volumes.
-* `private_access_settings_id` - (Optional, AWS only) Canonical unique identifier of [databricks_mws_private_access_settings](mws_private_access_settings.md) in Databricks Account
+* `network_id` - Modifying [networks on running workspaces](mws_networks.md#modifying-networks-on-running-workspaces) would require three separate `terraform apply` steps.
+* `credentials_id`
+* `storage_customer_managed_key_id`
+* `private_access_settings_id`
```

Comment:
Too much was removed?

Reply:
these already have descriptions above - this is just to highlight which parameter can be updated